### PR TITLE
Updates 3.7 RNs PR

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

File: modules/rn_3_7_0.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
+[[rn-3-700]]
+== Version 3.7.0
+
+=== quay / clair / quay-builder
+
+Added/Changed:
+
+* With {productname} 3.7, users have the ability to report storage consumption and to contain registry growth by establishing configured storage quota limits. Although there are no limitations on a Quay organization's storage consumption, on-premise Quay users are now equipped with quota reporting and quota management capabilities to manage the capacity limits of their environment. (link:https://issues.redhat.com/browse/PROJQUAY-302[PROJQUAY-302], link:https://issues.redhat.com/browse/PROJQUAY-253[PROJQUAY-253])
```
Comment (HammerMeetNail) [inline modules/rn_3_7_0.adoc:8]: > Although there are no limitations on a Quay organization's storage consumption

How are limitations on an organizations storage consumption different than managing the capacity limits? I think we're offering a guarantee that an org cannot exceed the quota limit if a policy is set to reject at a certain percentage. @kwestpharedhat  that sound right?

File: modules/rn_3_7_0.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+* The bare-metal constraint required to run builds has been removed by adding an additional build option which does not contain the virtual machine layer. As a result, builds can be run on virtualized platforms. Backwards compatibility to run previous build configurations are also available. (link:https://issues.redhat.com/browse/PROJQUAY-295[PROJQUAY-295])
+
+* {productname} can now act as a proxy cache to circumvent pull-rate limitations from upstream registries. This feature also accelerates pull performance, because images are pulled from the cache rather than upstream dependencies. Cached images are only updated when the upstream image digest differs from the cached image, reducing rate limitations and potential throttling. (link:https://issues.redhat.com/browse/PROJQUAY-465[PROJQUAY-465])
```
Comment (HammerMeetNail) [inline modules/rn_3_7_0.adoc:12]: > circumvent pull-rate limitations

nit: circumvent makes me think that we're giving users a way to surpass the rate limit...which is true for pulls of the same image but not if they try to pull an excessive number of different images. Thinking maybe `mitigate` is a better word that `circumvent`?

File: release_notes/master.adoc [hunk 3]
```diff
@@ -17,6 +17,7 @@ endif::downstream[]
 
 {productname}, version 3 is the latest major version.
 
+include::modules/rn_3_7_0.adoc[leveloffset=+1]
```
Comment (gabriel-rh) [inline release_notes/master.adoc:20]: Can you change the file name to rn_3_70 just to maintain the sequence?

File: modules/rn_3_70.adoc [hunk 4]
```diff
@@ -1,97 +1,69 @@
...
 
-* link:https://issues.redhat.com/browse/PROJQUAY-3030[PROJQUAY-3030]. As a Quay user I want to be able to proxy images through Quay orgs
+* Advanced Clair configuration is now available for {productname} 3.7. The following features are now available to Quay administrators on the Quay Operator:
 
-* link:https://issues.redhat.com/browse/PROJQUAY-3033[PROJQUAY-3033]. As a Quay user I want proxied images to be stored in Quay so that my pulls are faster
+** Configuration of Clair's updater set through the Quay Operator.
+** Configuration of the database connection string through the Quay Operator.
+** Configuration of custom certificates into the Clair deployment, which allows support of internal HTTPS proxies.
+** Override, or re-size, the size and storage class of the Postgres Database PVC.
+** Support for alternative fully qualified domain names (FQDN) for Clair that can leverage a global load balancing mechanism fronting different clusters running Clair.
```
Comment (ricardomaraschini) [inline modules/rn_3_70.adoc:38]: This is misleading or I did not understand it. There is no global load balancing for Clair.

File: modules/rn_3_70.adoc [hunk 5]
```diff
@@ -1,97 +1,69 @@
...
++
+For more information, see link:https://issues.redhat.com/browse/PROJQUAY-2110[PROJQUAY-2210].
 
+* With advanced Clair configuration, users can also provide a custom Clair configuration for an unmanaged Clair database on the {productname} Operator. An unmanaged Clair database allows the {productname} Operator to work in a Geo-Replicated environment, where multiple instances of the Operator must communicate with the same database. An unmanaged Clair database can also be used when a user requires a highly-available (HA) Clair database that exists outside of a cluster. (link:https://issues.redhat.com/browse/PROJQUAY-1696[PROJQUAY-1969])
```
Comment (ricardomaraschini) [inline modules/rn_3_70.adoc:42]: "where multiple instances of the Operator must communicate with the same database" is wrong, what communicates with the same database are the Clair instances (or Quay instances), not the operator.

File: modules/rn_3_70.adoc (hunk 4 above)
Comment (stevsmit) [reply modules/rn_3_70.adoc:38]: @ricardomaraschini should it be removed, or say

** Support for alternative fully qualified domain names (FQDN) for Clair that users can leverage a global load balancing mechanism fronting different clusters running Clair.

Comment (ricardomaraschini) [reply modules/rn_3_70.adoc:38]: Removed. There is no global load balancing support for Clair.

**Pull request merged**